website/content/docs/connect/registration/service-registration.mdx

```diff
@@ -1,13 +1,11 @@
 ---
 layout: docs
-page_title: Connect - Service Registration
+page_title: Service Mesh Proxy: Configuration
 description: >-
-  A per-service proxy sidecar transparently handles inbound and outbound service
-  connections. You can register these sidecars with reasonable defaults by nesting
-  their definitions in the service definition.
+Use service mesh proxy configuration files to define the parameters and behaviors of custom proxies deployed with services in your mesh. Learn about configuration options and how to format them with examples.
```

Comment:
This page is confusing (in my opinion), but isn't about custom proxies specifically. You can use this with Envoy proxies.
```suggestion
You can register a service mesh sidecar proxy separate from the registration of the service instance it fronts. Learn about proxy configuration options and how to format them with examples.
```